dataland-frontend/src/utils/Constants.ts

```diff
@@ -30,4 +30,7 @@ export const TIME_DELAY_BETWEEN_SUBMIT_AND_NEXT_ACTION_IN_MS = 2000;
 
 export const UPLOAD_FILE_SIZE_DISPLAY_DECIMALS = 2;
 
+// Same as standard for windows machines. File name can't have one of the special characters <>:"|?/\ and shouldn't end with a whitespace or .
```

Comment:
Shouldn't this be the minimum from what is accepted on any major OS? windows, macos, ubuntu/linux

Reply:
Add `*` to the list in the comment

Reply:
Apparently, MacOs and Linux only disallow very few characters in filenames which are already included in the ones forbidden by windows.

Reply:
I added the `*` to the comment.